### PR TITLE
Homework Day 02

### PR DESCRIPTION
**Pull request by dalalsa**
How difficult did you find this (out of 10)? 0 being no problems at all, 10 being impossible;
3
Was there anything that you struggled with?;
The Pluralizer function
Is there anything that you’d like some further information on?;
no
Roughly how long did it take?
1 and half hour 

Thanks

### CODE REVIEW

**Comment by dalalsa** [comment]
How difficult did you find this (out of 10)? 0 being no problems at all, 10 being impossible;
3
Was there anything that you struggled with?;
NO
Is there anything that you’d like some further information on?;
NO
Roughly how long did it take?
1 hour

File: Dalal_Saeed/week_01/day_02/Conditionals_LAB/main.js [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+//The World Translator
+var language = "English";
+if(language === "English"){
+    console.log("Hello World!");
+}
+else if (language === "Arabic"){
+    console.log("مرحبا بالعالم !");
+}
+else if (language === "Frensh"){
+    console.log("مرحبا بالعالم !");
```
Comment (mfalthaw) [inline Dalal_Saeed/week_01/day_02/Conditionals_LAB/main.js]: That's some fancy French :D

File: Dalal_Saeed/week_01/day_02/Conditionals_LAB/main.js [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+}
+if(userGrade >= 70){
+    "You Got an C"
+}
+if(userGrade >= 60){
+    "You Got an D"
+}
+if(userGrade < 60){
+    "You Got an F"
+}
```
Comment (mfalthaw) [inline Dalal_Saeed/week_01/day_02/Conditionals_LAB/main.js]: Remember to print the grade using `console.log()`.

File: Dalal_Saeed/week_01/day_02/Conditionals_LAB/main.js [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+else if (num > 1 ){
+    if (noun.endsWith("s")||noun.endsWith("x")|| noun.endsWith("z")||noun.endsWith("ch")||noun.endsWith("sh")){
+    console.log("You have "+ num +" "+noun+"es");}
+    else if (noun.endsWith("y")||noun.endsWith("y")||noun.endsWith("y")||noun.endsWith("y")){
+        if (noun.endsWith("ay")||noun.endsWith("ey")||noun.endsWith("oy")||noun.endsWith("iy")){
+        console.log("You have "+ num +" "+noun+"s");}
+        else {
+            console.log("You have "+ num +" "+noun+"ies");}
+        }
+    }
```
Comment (mfalthaw) [inline Dalal_Saeed/week_01/day_02/Conditionals_LAB/main.js]: Nice!

File: Dalal_Saeed/week_01/day_02/Variables_LAB/main.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+//The Geometrizer
+var radius = 60;
+console.log("The circumference is "+NN);
+console.log("The area is "+NN);
+
+//The Temperature Converter
+var celsiusTemperature = 37;
+console.log(NN+" °C "+ "is " + NN +" °F");
+var fahrenheitTemperature = 37;
+console.log(NN+" °F "+ "is " + NN +" °C");
```
Comment (mfalthaw) [inline Dalal_Saeed/week_01/day_02/Variables_LAB/main.js]: Almost done with this one, 1-2 more steps and you're set!

File: Dalal_Saeed/week_01/day_02/Conditionals_LAB/main.js (hunk 1 above)
Comment (dalalsa) [reply Dalal_Saeed/week_01/day_02/Conditionals_LAB/main.js]: uh oh !

**Pull request merged**